python/yb/download_and_extract_archive.py

```diff
@@ -191,7 +191,6 @@ def download_and_extract(url, dest_dir_parent, local_cache_dir, nfs_cache_dir):
             logging.info("Failed creating directory '%s': %s", local_cache_dir, ex)
 
     check_dir_exists_and_is_writable(local_cache_dir, "Local cache")
-    check_dir_exists_and_is_writable(nfs_cache_dir, "NFS cache")
```

Comment:
With this change, `./yb_build.sh tsan --clean --dltp` was able to make progress. Thanks!